python/cuml/ensemble/randomforestregressor.pyx

```diff
@@ -485,12 +508,14 @@ class RandomForestRegressor(Base):
         concat_model_handle = concatenate_trees(deref(model_handles))
 
         concat_model_ptr = <size_t> concat_model_handle
-        return ctypes.c_void_p(concat_model_ptr).value
-
-    def _concatenate_model_bytes(self, concat_model_handle):
-        cdef uintptr_t model_ptr = <uintptr_t> concat_model_handle
-        concat_model_bytes = save_model(<ModelHandle> model_ptr)
-        self.model_pbuf_bytes = concat_model_bytes
+        self.concat_handle = ctypes.c_void_p(concat_model_ptr).value
+        cdef uintptr_t model_ptr = <uintptr_t> self.concat_handle
+        cdef vector[unsigned char] pbuf_mod_info = \
+            save_model(<ModelHandle> model_ptr)
+        cdef unsigned char[::1] pbuf_mod_view = \
+            <unsigned char[:pbuf_mod_info.size():1]>pbuf_mod_info.data()
+        self.concat_model_bytes = memoryview(pbuf_mod_view).tobytes()
```

Comment:
Since it sounds like we want `concat_model_bytes` to hold onto mutable memory (`bytes` objects are immutable), would make sense to go to `bytearray` here. Though please feel free to push back if I'm just not understanding the workflow here 🙂

```suggestion
        self.concat_model_bytes = bytearray(memoryview(pbuf_mod_view))
```